nixos/modules/image/repart-image.nix

```diff
@@ -0,0 +1,75 @@
+{ name
+, fileSystems
+, partitions
+, split
+, seed
+, definitionsDirectory
```

Comment:
Can you please separate the "arguments" from the packages? E.g. by moving the arguments to the bottom and adding a comment saying something like "arguments".

Reply:
added